### PR TITLE
Added "make_mvc.mak" file to "lang"

### PR DESCRIPTION
**Pull request by RestorerZ**
1. To convert menu files to Windows, the file "make_mvc.mak" has been added.
2. The list of files to be converted has been moved to the "make_all.mak" file.
3. The rule "menu_slovak_slovak_republic.1250.vim: menu_sl_si.utf-8.vim" is commented out in the Makefile file, because these are menu files for different languages.

The file "menu_slovak_slovak_slovak_republic.1250.vim" is the original file for Slovak language.
The file "menu_sl_si.utf-8.vim" is the original file for Slovenian language.
There is also a rule "menu_sk_sk.iso_8859-2.vim: menu_slovak_slovak_republic.1250.vim", which converts the original file to ISO-8859-2 encoding.

### CODE REVIEW

**Comment by k-takata** [comment]
Please rename `make_*.mak` to `Make_*.mak`. (See also the makefiles in `src` and `src/testdir`.)

**Comment by RestorerZ** [comment]
> Please rename `make_*.mak` to `Make_*.mak`. (See also the makefiles in `src` and `src/testdir`.)

Done.

File: runtime/lang/Make_all.mak [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+		menu_sl_si.cp1250.vim \
+		menu_sl_si.latin2.vim \
+		menu_sr_rs.ascii.vim \
+		menu_sr_rs.iso_8859-2.vim \
+		menu_sr_rs.iso_8859-5.vim \
+		menu_tr_tr.cp1254.vim \
+		menu_tr_tr.iso_8859-9.vim \
+		menu_uk_ua.cp1251.vim \
+		menu_uk_ua.koi8-u.vim \
+
```
Comment (chrisbra) [inline runtime/lang/Make_all.mak:29]: can you please remove those additional empty lines? and the commented out line `menu_slovak_slovak_republic`?

Comment (RestorerZ) [reply runtime/lang/Make_all.mak:29]: @chrisbra 
Yes, I deleted the blank lines. And the commented out line as well.
But what about the rule "menu_slovak_slovak_republic.1250.vim: menu_sl_si.utf-8.vim"? I commented it out, but is it necessary to leave it in?
I wrote to @adaext , but didn't get a reply. Is it possible that it meant something else? No matter how many times I tried, I couldn't find what it should be.

Comment (RestorerZ) [reply runtime/lang/Make_all.mak:29]: removed the rule "menu_slovak_slovak_republic.1250.vim: menu_sl_si.utf-8.vim"

**Comment by chrisbra** [comment]
thanks!

**Pull request merged**